runtime/near-evm-runner/Cargo.toml

```diff
@@ -20,15 +20,15 @@ enum-primitive-derive = "0.2"
 sha2 = ">=0.8,<0.10"
 sha3 = "0.8"
 rlp = "0.4.2"
-keccak-hash = "0.2.0"
+keccak-hash = "0.4.1"
 ripemd160 = "0.9.0"
 libsecp256k1 = "0.3.5"
 
-evm = { git = "https://github.com/openethereum/openethereum", rev = "v2.6.8" }
-vm = { git = "https://github.com/openethereum/openethereum", rev = "v2.6.8" }
-bn = { git = "https://github.com/paritytech/bn", default-features = false }
+evm = { git = "https://github.com/openethereum/openethereum", rev = "v2.7.2" }
+vm = { git = "https://github.com/openethereum/openethereum", rev = "v2.7.2" }
+bn = { git = "https://github.com/near/bn", branch = "upgrade-rand" }
```

Comment:
cargo didn't handle cache branch well (e.g. push a new commit to branch, if you `rm -rf ~/.cargo/git` and build it pick new commit, otherwise it still use old commit), I suggest use a commit hash.

Reply:
Thanks. Done in 52590d8a05143bc5ff6c7db0a65c5ccf222fcb44.